scavenger-api/build.gradle.kts

```diff
@@ -37,6 +37,7 @@ dependencies {
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
     implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310")
     implementation("mysql:mysql-connector-java")
+    implementation("org.apache.commons:commons-csv:1.10.0")
```

Comment:
How about using kotlin-csv?
 https://github.com/doyaaaaaken/kotlin-csv

Reply:
I fixed it on commits cf5fbe8, 284ca5d, and 1e29eb1. Please review these commits.